### ISSUE TITLE
keybind to code

### ISSUE THREAD

**Issue by kyak** [labwc/labwc#1069]
It would be great to have keybinds support keycodes.

Currently, I would have to create two keybinds for two keyboard layouts. This is because keybind like e.g. `W-l` will not work when another keyboard layout is active - I would have to create the second keybind like `W-д`.

This used to work in openbox, i.e. I only had to specify keybind for `W-l`. It also works for sway by using `--to-code` with `bindsym`.

Can we support it in labwc?

**Comment by Consolatis** [comment]
We should support that already, we default to checking the translated keysym but fall back to raw keysyms if the translated one didn't match. Thus a keybind like `W-1` should actually work in a somewhat recent version.

**Comment by Consolatis** [comment]
Right.. for the fallback we only look at keys and ignore their modifiers (thus `W-S-2` should work on all layouts). But for the letters the modifier don't play a role. Hrm.

**Comment by Consolatis** [comment]
Implemented by
- #1072

Feel free to point out any issues that may arise.
